bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -37,21 +37,6 @@ spec:
       hostNetwork: true
       hostPID: true
       priorityClassName: system-node-critical
-      initContainers:
```

Comment:
so, wait, how are the sdn binaries installed now?

Reply:
By `network-plugin-install`